presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -112,6 +113,8 @@ fun EventScreen(
 
                     Spacer(modifier = modifier.height(26.dp))
 
+                    val context: Context = LocalContext.current
```

Comment:
@philipplackner @MKiperszmid Is it ok to have this context declared here. As I need to pass it to the `setupAlarmReminderItem`  to obtain the string from the resource id?

Reply:
It´s fine to get the context so you can use stringResources